capstone-backend/src/main/java/com/google/sps/servlets/Utility.java

```diff
@@ -213,21 +234,18 @@ public static JsonObject createRequestBodyJson(HttpServletRequest request) throw
     }
     if (request.getParameterMap().size() == 0) {
       asyncQuery = collectionReference.get();
-    }
-    else {
+    } else {
       asyncQuery = getByField(collectionReference, request, response, genericClass);
       if (asyncQuery == null) {
         return null;
       }
     }
     try {
-      retrievedObjects = asyncQuery.get().getDocuments().stream()
-                             .map(d -> {
-                                        T t = d.toObject(genericClass.getMyType());
-                                        t.setId(d.getId());
-                                        return t;
-                                       })
-                             .collect(Collectors.toList());
+      retrievedObjects = asyncQuery.get().getDocuments().stream().map(d -> {
+        T t = d.toObject(genericClass.getMyType());
```

Comment:
Should the map function be indented from asyncQuery instead of from retrievedObjects?